lib/pbench/server/api/resources/__init__.py

```diff
@@ -838,13 +844,15 @@ class ApiSchema:
 
     def __init__(
         self,
-        method: API_METHOD,
-        operation: API_OPERATION,
+        method: ApiMethod,
+        operation: OperationCode,
```

Comment:
Yeah, but `POST` is currently overloaded, and it's impossible to disambiguate without intimate knowledge of the subclass behavior. This lets the subclass provide that knowledge. I didn't see any other practical way to acquire it.

Reply:
Yeah, I wasn't so much suggesting that we could get rid of the `operation` parameter as that the function could provide a default value for it, which would allow most of the callers (i.e., non-`POST` operations) to omit the parameter from their calls.  As you say, many of the `POST` operations would still need to specify it explicitly.

Unfortunately, the logic required is somewhat complex, so I'm not sure it's worth it, but I figured I would mention the possibility, nonetheless.